src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx

```diff
@@ -61,16 +76,15 @@ export const MeetTheSpecialists: React.FC = () => {
       <Text mb={2} variant={["xs", "sm"]}>
         Our specialists span today’s most popular collecting categories.
       </Text>
-      <Rail
-        title=""
-        getItems={() => {
-          return SPECIALISTS.map(i => (
+      <Shelf showProgress={false}>
```

Comment:
Replacing `Rail` with `Shelf` because Rails is the same as Shelf but with a title, and we don't need a title 👍